diff --git a/lua/barbar/events.lua b/lua/barbar/events.lua
index 076f18b..8f9cd37 100644
--- a/lua/barbar/events.lua
+++ b/lua/barbar/events.lua
@@ -275,7 +275,7 @@ function events.enable()
 
       -- we want the offset to begin ON the first win separator
       -- WARN: don't use `win_separator` here
-      offset = offset - 1
+      -- offset = offset - 1
 
       -- width cannot be less than zero
       return max(0, offset)
